examples/batch-proposal/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "example-batch-proposal",
-  "version": "1.46.0",
+  "version": "1.47.0-rc.0",
   "private": true,
   "main": "index.js",
   "author": "Ernesto García <mverzilli@openzeppelin.com>",
```

Comment:
Just noticed this dual identity @ernestognw 😆

Reply:
You gotta see me with my hair tied up. @dylankilkenny can confirm